src/app/models/types/responses/index.ts

```diff
@@ -0,0 +1 @@
+export * from '../responses/product-query-args.type';
```

Comment:
Не нужно выходить из папки. Ты получается тут выходишь из папки responses и обратно заходишь.
оставь просто 
```suggestion
export * from './product-query-args.type';
```